CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# v8.1.0 (2025-05-05)
+
+## OS Changes
+* Add containerd 2.0 package ([#485])
+
+## Orchestrator Changes
+### Kubernetes
+* Add settings `container-log-max-workers`, `container-log-monitor-interval` and `single-process-oom-kill` ([#487], [#489])
```

Comment:
Referring to our past [changelog](https://github.com/bottlerocket-os/bottlerocket/blob/develop/CHANGELOG.md?plain=1#L1332-L1337)

 ```suggestion
* Add support for more Kubernetes Settings ([#487], [#489])
  * containerLogMaxWorkers
  * containerLogMonitorInterval
  * singleProcessOOMKill
```